src/Plugins/BotSharp.Plugin.RoutingSpeeder/Controllers/RoutingSpeederController.cs

```diff
@@ -24,9 +25,17 @@ public RoutingSpeederController(IServiceProvider service)
     public IActionResult TrainIntentClassifier(TrainingParams trainingParams)
     {
         var intentClassifier = _service.GetRequiredService<IntentClassifier>();
-        intentClassifier.InitClassifer(trainingParams.Inference);
+        // intentClassifier.InitClassifer(trainingParams.Inference);
         intentClassifier.Train(trainingParams);
         return Ok(intentClassifier.Labels);
     }
 
+    [HttpPost("/routing-speeder/classifier/inference")]
```

Comment:
Is this API for train or inference?
The function name doesn't match with endpoint.